src/helics/common/JsonProcessingFunctions.cpp

```diff
@@ -16,15 +16,34 @@ SPDX-License-Identifier: BSD-3-Clause
 
 namespace helics::fileops {
 
-bool looksLikeCommandLine(std::string_view testString)
+bool looksLikeJson(std::string_view jsonString)
 {
-    if (testString.empty()) {
+    if (jsonString.empty()) {
         return false;
     }
-    if (testString.front() == '-') {
+    if ((jsonString.front() == '{') || (jsonString.front() == '//' && jsonString.back() == '}')) {
```

Comment:
I don't think this check is sufficient to determine if a file looks like JSON. Based on this, a C/C++ file such as this would look like JSON:

```c
// Copyright blah

void main() {
}
```

Reply:
I was struggling with this a bit.  I need a very quick check that works most of the time.  If there was a very fast reliable way to check for valid json I would be interested

Reply:
If the scope is specifically a valid JSON config string, if we can guarantee that the outer most element is a JSON object ("{...}"), then I think the fastest check would strip leading whitespace and then check if the line begins with `{`. If it begins with a `//` for a comment line, do the same check for the next line.

I guess the ending of the file could also have comment lines that need to be ignored to find the last `}`.

If those two criteria don't hold, I think we can say with certainty that it is not going to be a valid JSON config string. If they are true, then we could probably skip the rest of the checks.

Reply:
In the best case, I think that check would be equivalent to the current condition given here -- worst case it turns into something closer to the subsequent checks that look for the exact locations of the brackets.